_posts/2013-04-11-testing.md

```diff
@@ -52,28 +52,28 @@ All Ember Apps come with built-in [ember test helpers](http://emberjs.com/guides
 which are useful for writing integration tests.
 In order to use them, you will need to import `tests/helpers/start-app.js`, which injects the required helpers.
 
-Be sure to use the injected `module` function to invoke `setup` and `teardown`.
+Be sure to use the `module` function to invoke `beforeEach` and `afterEach`.
 
 {% highlight javascript linenos %}
 import Ember from "ember";
-import { test } from 'ember-qunit';
+import { module, test } from 'qunit';
```

Comment:
@rwjblue updated again - thanks for the clarification. I hope this line is now as it should be? Was the previous `import { test } from 'ember-qunit'` incorrect or would both work?